chart/customer/templates/istio_customer.yaml

```diff
@@ -52,4 +52,16 @@ spec:
         host: {{ .Release.Name }}-{{ .Values.service.name }}.{{ .Release.Namespace }}.svc.cluster.local
         port:
           number: {{ .Values.service.servicePort }}
+    retries:
+      attempts: 4
+      retryOn: 4xx,5xx
```

Comment:
`4xx` isn't a valid code. There is `retriable-4xx` in the docs, but doesn't seem to be too useful.

We'll just stick with only `5xx`